go.mod

```diff
@@ -3,12 +3,10 @@ module github.com/openshift/client-go
 go 1.13
 
 require (
-	github.com/openshift/api v0.0.0-20200116145750-0e2ff1e215dd
+	github.com/openshift/api v0.0.0-20200311151921-fdf269f98861
 	github.com/spf13/pflag v1.0.5
 	k8s.io/api v0.18.0-beta.2
 	k8s.io/apimachinery v0.18.0-beta.2
 	k8s.io/client-go v0.18.0-beta.2
 	k8s.io/code-generator v0.18.0-beta.2
 )
-
-replace github.com/openshift/api => github.com/openshift/api v0.0.0-20200306192528-e5737622441f
```

Comment:
why is this gone?

Reply:
Not needed, it pins the right version above.